config/custom-environment-variables.yaml

```diff
@@ -112,3 +112,9 @@ ecosystem:
     store: ECOSYSTEM_STORE
     # Badge service (needs to add a status and color)
     badges: ECOSYSTEM_BADGES
+    # List of module names, or objects { name, config } for instantiation to use in sd-setup
+    setup:
+        - sd-scm
+        - SD_SETUP_PLUGINS
```

Comment:
If SD_SETUP_PLUGINS is set to "my_custom_plugin", this seems like it would work. Are we only allowing a single plugin to be added?